docs/explanation/e-hardening.md

```diff
@@ -0,0 +1,119 @@
+|-----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| OpenStack | [OpenStack Security Guide](https://docs.openstack.org/security-guide/)                                                                                                                                                                                        |
+| AWS       | [Best Practices for Security, Identity and Compliance](https://aws.amazon.com/architecture/security-identity-compliance), [AWS security credentials](https://docs.aws.amazon.com/IAM/latest/UserGuide/security-creds.html#access-keys-and-secret-access-keys) | 
+| Azure     | [Azure security best practices and patterns](https://learn.microsoft.com/en-us/azure/security/fundamentals/best-practices-and-patterns), [Managed identities for Azure resource](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/)                                                                            |
+
+### Juju 
+
+Juju is the component responsible for orchestrating the entire lifecycle, from deployment to Day 2 operations, of 
+all applications. Therefore, it is imperative that it is set up securely. Please refer to the Juju documentation for more information on:
+
+* [Juju security](https://discourse.charmhub.io/t/juju-security/15684)
+* [How to harder your deployment](https://juju.is/docs/juju/harden-your-deployment)
+
+#### Cloud credentials
+
+When configuring the cloud credentials to be used with Juju, ensure that the users have correct permissions to operate at the required level. 
+Juju superusers responsible for bootstrapping and managing controllers require elevated permissions to manage several kind of resources, such as
+virtual machines, networks, storages, etc. Please refer to the references below for more information on the policies required to be used depending on the cloud. 
+
+| Cloud     | Cloud user policies                                                                                                                                                                                                                            |
+|-----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| OpenStack |                                                                                                                                                                                                                                                |
```

Comment:
Can we add something for the OpenStack?

Reply:
I had already asked a person in openstack for such information, but he hasn't gotten back to me with this yet. As soon as I have this info, I will update. However there is also the chance that that information may not exists. What should we do in that case? Just putting "N/A"?

Reply:
Yes, `TBA`, `TBD`, or `N/A` is much better than an empty cell.